src/main/java/com/wishlist/model/Category.java

```diff
@@ -0,0 +1,25 @@
+package com.wishlist.model;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import com.wishlist.enums.Categories;
+import javax.persistence.*;
+import java.util.Collection;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name="categories")
+
+public class Category {
+
+    @Id
+    Long id;
+    @ElementCollection(targetClass = Categories.class)
+    @Enumerated(EnumType.STRING)
+    Collection<Categories> categories;
```

Comment:
Quite generic type. IDK what is the implementation underneath and what is the performance impact. Probably uses HashSet.